### PR TITLE
Release notes for 1.7.5

### PR DESCRIPTION
**Pull request by JimmyCYJ**
[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (shamsher31) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:20]:
```suggestion
- **Fixed** how `install-cni` applies `istio-cni` plugin configuration, `install-cni` will now remove existing `istio-cni` plugins from the CNI config before inserting `istio-cni` plugin configuration (new behavior) rather than just append a new configuration to the existing list (former behavior). ([Issue #27771](https://github.com/istio/istio/issues/27771))
```

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (shamsher31) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:28]:
```suggestion
- **Fixed** an issue causing a short spike in errors during in place upgrades from Istio 1.6 to Istio 1.7. As a result of this fix, users who already have Istio 1.7 deployed but still have proxies left on version 1.6 will see a similar spike during this upgrade. It is highly recommended you either migrate all existing proxies to version 1.7 prior to this release. Alternatively, to retain the previous behavior, you may set the `PILOT_ENABLE_TLS_XDS_DYNAMIC_TYPES=false` environment variable in Istiod. ([Issue #28120](https://github.com/istio/istio/issues/28120))
```

**Review by jwendell — COMMENTED** [review]
I believe https://github.com/istio/istio/pull/28845#issuecomment-728639548 is missing

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (ericvn) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:20]: Split the sentence after plugin configuration.

Not sure the (new behavior) and (old behavior) are needed.

Also will ~~now~~ remove and rather than ~~just~~ append

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (ericvn) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:22]: Split sentence after expansion scenario). 

Still not sure on the new/former behavior in parends.

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (ericvn) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:28]: c/you/users/

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (brian-avery) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:20]: This might be better: 

```
- **Fixed** how `install-cni` applies `istio-cni` plugin configuration. Previously, new configurations would be appended to the list. This has been changed to remove existing `istio-cni` plugins from the CNI config before inserting new plugins. 
```

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (brian-avery) [inline content/en/news/releases/1.7.x/announcing-1.7.5/index.md:24]: Reading this I'm not sure if this has been fixed not to run the secret fetcher when proxy is configured or if it was improperly running the secret fetcher before and now no longer is.

Comment (brian-avery) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:24]: Looking at https://github.com/istio/istio/pull/26978, it looks like this changes it not to run the secret fetcher with file mounted certs.

Comment (brian-avery) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:24]:
```suggestion
- **Fixed** Istio to not run gateway secret fetcher when proxy is configured with `FILE_MOUNTED_CERTS`.
```

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (brian-avery) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:28]: This release note is confusing to me. It talks about reducing errors, but not what was fixed to reduce those errors. It then talks about how you can retain the previous behavior. What is the previous behavior? A bunch of errors occurring?

Comment (brian-avery) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md:28]:
```suggestion
- **Fixed** an issue causing a short spike of errors during in place upgrades from Istio 1.6 to 1.7. Previously, the TLS version would be upgraded automatically from TLSv2 to TLSv3. This caused a spike in errors if there was a mix of Istio 1.6 and Istio 1.7 proxies running. The `PILOT_ENABLE_TLS_XDS_DYNAMIC_TYPES`=false environment variable was added to disable automatic TLS version updates. This is enabled by default but should be disabled for upgrades from Istio 1.6 to Istio 1.7. ([Issue #28120](https://github.com/istio/istio/issues/28120))
```

@howardjohn can you please verify my suggestion?

**Comment by JimmyCYJ** [comment]
> I believe [istio/istio#28845 (comment)](https://github.com/istio/istio/pull/28845#issuecomment-728639548) is missing

I have added one for that. Thanks.

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (JimmyCYJ) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md]: Thanks! I have updated this one.

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (JimmyCYJ) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md]: I have removed new/former behavior from this line.

File: content/en/news/releases/1.7.x/announcing-1.7.5/index.md
Comment (JimmyCYJ) [reply content/en/news/releases/1.7.x/announcing-1.7.5/index.md]: Done. Thanks!

**Review by jwendell — APPROVED** [review]

**Review by ericvn — COMMENTED** [review]
lgtm. Will wait on review from @brian-avery before merging.

**Review by brian-avery — APPROVED** [review]

**Review by ericvn — APPROVED** [review]

**Pull request merged**
